f5_openstack_agent/lbaasv2/drivers/bigip/agent_manager.py

```diff
@@ -440,6 +440,25 @@ def periodic_resync(self, context):
             # Resync the next time around.
             self.needs_resync = True
 
+    @periodic_task.periodic_task(spacing=30)
```

Comment:
The spacing here seems a little too frequent; however, instead of pushing stats to neutron, maybe we should just pull from the Neutron API.  Only populate stats when a call to get_loadbalancer_statistics is made.